crates/worker/src/partition/state_machine/tests/idempotency.rs

```diff
@@ -172,6 +172,7 @@ async fn complete_already_completed_invocation() {
         &invocation_id,
         InvocationStatus::Completed(CompletedInvocation {
             invocation_target: invocation_target.clone(),
+            span_context: Default::default(),
```

Comment:
Instead of using `Default::default` it is a bit more expressive to name the type one is creating a default value for: `Option::default`.

Reply:
TBH I find it redudant next to the field name. I understand when you call a method, but when building a struct where you have the field name aside sounds very redundant.